iib/workers/tasks/utils.py

```diff
@@ -1,27 +1,30 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 import base64
-from contextlib import contextmanager
 import functools
 import inspect
 import json
 import logging
 import os
 import re
 import subprocess
+from contextlib import contextmanager
```

Comment:
why you moved it here?

Reply:
I did not have to, I used "Optimize imports" from PyCharm. On the other hand I think it should belong to that place.  Because I am used to have first imports starting with `import`, then `from ... import ...`. And also try to preserve order suggested in [PEP8](https://www.python.org/dev/peps/pep-0008/#imports): 
```
Standard library imports.
Related third party imports.
Local application/library specific imports.
```

Reply:
yes and to follow that order
from contextlib import contextmanager 
should be first

Reply:
It is order like that, separate to groups just `import foo` comes before `from foo import bar`. Which, at least for me, has it logical meaning. You have another section of imports grouped and visually separated. It is better for reading the code. 

However I can revert that change if it is not according our code style.

Reply:
If you want to change the order of imports, please make a separate PR.